gateway/implementation/src/main/java/eu/openg/guessnumberapi/gateway/implementation/PostgresqlGameRepo.java

```diff
@@ -0,0 +1,109 @@
+package eu.openg.guessnumberapi.gateway.implementation;
+
+import eu.openg.guessnumberapi.domain.Game;
+import eu.openg.guessnumberapi.gateway.api.GameRepository;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.*;
+
+public class PostgresqlGameRepo implements GameRepository {
+    private static final Logger LOGGER = LogManager.getLogger(PostgresqlGameRepo.class);
+    private Connection con;
+
+    public PostgresqlGameRepo() {
+        connectToPostgresqlDatabase();
+        createPostgresqlTable();
+    }
+
+    private void connectToPostgresqlDatabase() {
+        con = null;
+        try {
+            Class.forName("org.postgresql.Driver");
+            con = DriverManager
+                    .getConnection("jdbc:postgresql://localhost:5432/postgres",
+                            "postgres", "guessnumber");
+            LOGGER.info("PostgreSql database opened successfully");
+        } catch (Exception e) {
+            LOGGER.error("Connection to PostgreSql database failed. " + e.getMessage());
+        }
+    }
+
+    private void createPostgresqlTable() {
+        try {
+            Statement stmt = con.createStatement();
```

Comment:
biški apie namingą reiktų paskaityt. mėtosi labai paprasta knyga lentynoj: The art of readable code, ten skyrius yra apie namingą nedidelis, per vakarą perskaitytum